events/equeue/tests/tests.c

```diff
@@ -687,6 +707,45 @@ void multithreaded_barrage_test(int N) {
     equeue_destroy(&q);
 }
 
+struct sCaQ
+{
+    int p;
+    equeue_t* q;
+};
+
+typedef struct sCaQ CountAndQueue;
+
+void simple_breaker(void *p) {
+	CountAndQueue* caq = (CountAndQueue*)p;
+	equeue_break(caq->q);
+    usleep(10000);
```

Comment:
nit: It looks like there's a mixture of tabs + spaces here. Can you change this to just 4 spaces for indention.